MeetupPlatformApi/Features/Meetups/DeleteMeetup/DeleteMeetupFeature.cs

```diff
@@ -29,6 +32,11 @@ public async Task<IActionResult> DeleteMeetup([FromRoute] Guid id)
             return NotFound();
         }
 
+        if (meetup.OrganizerId != CurrentUser.UserId)
+        {
+            return Forbid();
```

Comment:
You return here, but this method does not [ProducesResponseType(StatusCodes.Status403Forbidden)]